### PR TITLE
(#86) Commands: Support `g` suffix

### LINKED ISSUE #86: Support `g` suffix for gibibyte

### PR DESCRIPTION
**Pull request by gustav-langer**
This closes #86 by adding the ability to use a `g` suffix for Gibiytes.

### CODE REVIEW

**Comment by gustav-langer** [comment]
Please add `hacktoberfest-accepted` as a label to this pr :)

**Comment by ForNeVeR** [comment]
I think everything is considered as accepted by default if I merge it, provided the repo has `hacktoberfest` tag?

I think it worked that way for myself, at least 😅 

Anyway, let's check after I merge this if it will accept the PR or not. I am okay with adding a label if it helps you to get Hacktoberfest rating :)

**Review by ForNeVeR — APPROVED** [review]
Thanks for your contribution!

**Comment by gustav-langer** [comment]
> I think everything is considered as accepted by default if I merge it, provided the repo has `hacktoberfest` tag?

You're right, it already counts. Thanks!

**Pull request merged**
